rosdep/base.yaml

```diff
@@ -226,6 +226,10 @@ beep:
   fedora: [beep]
   gentoo: [app-misc/beep]
   ubuntu: [beep]
+benchmark:
+  debian: [libbenchmark-dev]
```

Comment:
Looking at this further, it looks like this is only supported on buster and later (and stretch-backports, but we don't usually allow those as rosdep keys).  Given that stretch is still supported (and still used by kinetic), we want to make sure we don't give the impression that it is available there.  So this would become:

```
debian:
  stretch: null
  *: [libbenchmark-dev]
```

Reply:
I didn't know about that syntax. Actually, looking at other places in the file it seems that `*` should be quoted: `'*'`

Today I learnt something! :smile: 

Changed.